runtime/doc/lua-guide.txt

```diff
@@ -386,6 +386,9 @@ window is used:
 >lua
     vim.bo[4].expandtab = true -- sets expandtab to true in buffer 4
     vim.wo.number = true       -- sets number to true in current window
+    vim.wo[0].number = true    -- same as above (and |:set|)
```

Comment:
```suggestion
    vim.wo[0].number = true    -- same as above (same as |:set|)
```

or 

```suggestion
    vim.wo[0].number = true    -- same as above 
```

Reply:
Either is fine with me (it's your PR, just make the change).

Since you ask, I think it's better to not mix up the Lua API presentation and the Vimscript API. If(!) we touch on global-local window options at all here, line 366 above would be the better place. (Or just give a reference to `|vim.wo|` where a full explanation can be found.)

Reply:
I agree.